unfoldingWord/translationCore
# Bugfix bspidel 2938

#### This pull request addresses:

Display text when no projects have been selected

#### How to test this pull request:
- First test showing projects. Should see project cards
- Delete all projects from user account 
 - on windows machine delete all files under
   - C:\\users\Owner\translationCore\Projects 
- Now show projects. Should see single line of text under Projects


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3212)
